cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/common/tools/ExceptionHandler.java

```diff
@@ -95,4 +100,35 @@ private static void openInvalidConfigurationErrorDialog(InvalidConfigurationExce
     });
   }
 
+  /**
+   * Opens up a message dialog for displaying further guidance on upgrading old templates.
+   *
+   * @param e {@link InvalidConfigurationException} occurred
+   */
+  private static void openMonolithicTemplatesErrorDialog(DeprecatedMonolithicTemplatesException e) {
+
+    PlatformUIUtil.getWorkbench().getDisplay().syncExec(() -> {
+      MessageDialog dialog = new MessageDialog(Display.getDefault().getActiveShell(), "Warning!", null,
+          e.getMessage() + " Further Information can be found at:" + WikiConstants.WIKI_UPGRADE_MONOLITHIC_TEMPLATES,
+          MessageDialog.ERROR, new String[] { "Upgrade", "Postpone", "Cancel" }, 2);
+      dialog.setBlockOnOpen(true);
+
+      int result = dialog.open();
+      if (result == 0) {
+        // TODO Integrate the Upgrader from #1502
+        // For now Call HealthCheck
+        // new HealthCheckDialog().execute();
+        // new UpgradeTemplatesDialog().execute();
```

Comment:
Let's wait for the upgrader pr to finalize here

Reply:
I think this then should be done in #1518 
Is that ok to leave the TODO there? So then when we begin with the next ticket, one can see where to start.